pennylane_forest/converter.py

```diff
@@ -0,0 +1,626 @@
+import warnings
+from collections.abc import Sequence
+
+import numpy as np
+import pennylane as qml
+import pennylane_forest as plf
+import pyquil
+import pyquil.gates as g
```

Comment:
I have no idea, I just wanted to keep it short

Reply:
:laughing:

Reply:
I actually wanted to keep the gates separate to avoid name clashes, because pennylane also has gates like `CNOT`, but this is only relevant in the tests now, so I applied your suggestion!